### PR TITLE
Processing image

### PR DESCRIPTION
**Pull request by william-rgt**

### CODE REVIEW

File: lib/da_funk/helper.rb [hunk 1]
```diff
@@ -334,6 +334,18 @@ def rjust(string, size, new_string)
       end
     end
 
+    def processing(image, message = {})
```
Comment (scalone) [inline lib/da_funk/helper.rb]: The processing method is not part DaFunk::Helper domain. @william-rgt could you remove it from here, please.

File: lib/da_funk/helper.rb [hunk 2]
```diff
@@ -334,6 +334,18 @@ def rjust(string, size, new_string)
...
+      Device::Display.print_bitmap(to_bmp(image)) if bmp_exists?(image)
+    end
+
+    def bmp_exists?(bmp)
+      File.exists?(to_bmp(bmp))
+    end
+
+    def to_bmp(image)
```
Comment (scalone) [inline lib/da_funk/helper.rb]: @william-rgt it's important to check if the image path already have an extension.

File: lib/da_funk/helper.rb (hunk 1 above)
Comment (william-rgt) [reply lib/da_funk/helper.rb]: @scalone Ok. but, Where should we implement this?

File: lib/da_funk/helper.rb (hunk 2 above)
Comment (william-rgt) [reply lib/da_funk/helper.rb]: ok.

**Pull request merged**